test/e2e/diagnose.go

```diff
@@ -82,7 +83,24 @@ func checkUnschedulablePods(clients *test.Clients, logger *logging.BaseLogger) {
 		}
 	}
 	if unschedulablePods != 0 {
-		logger.Errorf(fmt.Sprintf("%v out of %v pods are unschedulable. insufficient cluster capacity?",
-			unschedulablePods, totalPods))
+		logger.Errorf("%v out of %v pods are unschedulable. insufficient cluster capacity?", unschedulablePods, totalPods)
+	}
+}
+
+func checkControllers(clients *test.Clients, logger *logging.BaseLogger) {
+	for _, name := range []string{
+		"activator",
+		"autoscaler",
+		"controller",
+		"webhook",
+	} {
+		dep, err := clients.KubeClient.Kube.AppsV1().Deployments("knative-serving").Get(name, metav1.GetOptions{})
+		if err != nil {
+			logger.Errorf("Unable to get %v deployment: %v", name, err)
+			continue
+		}
+		if dep.Status.AvailableReplicas != 1 {
```

Comment:
This will need some adjustment in #2171. @mattmoor fyi

Reply:
`activator` is not a controller, neither is `webhook`.  Can we just remove this?